src/main/java/org/opentripplanner/netex/mapping/calendar/CalendarServiceBuilder.java

```diff
@@ -64,6 +66,13 @@ public Collection<ServiceCalendarDate> createServiceCalendar() {
     return dates;
   }
 
+  public ServiceCalendar getEmptyCalendar() {
+    ServiceCalendar emptyCalendar = new ServiceCalendar();
+    emptyCalendar.setServiceId(EMPTY_SERVICE_ID);
+    emptyCalendar.setPeriod(ServiceDateInterval.unbounded());
```

Comment:
Rename `getEmptyCalendar()` to `createEmptyCalendar()`.

Reply:
Done